### PR TITLE
DP-31793 header alerts

### PR DESCRIPTION
**Pull request by ygannett**
**Description:**
The mass_alert module is modified for the icon change for "notice."

**Jira:** (Skip unless you are MA staff)
DP-31793

**To Test:**
Sample page:  /how-to/qag-request-help-with-a-computer-problem
The sample page only has an item for "alert."  Need to edit the item to "notice" or add another alert content with "notice."
- [ ] Find the component has a gray border.
- [ ] Find the border doesn't add up (= get thicker) when it has multiple alert items.
- [ ] The icon for the notice is charged to the bell and it's green.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Changelog is not valid the following item(s) needs to be fixed (data should NOT have additional properties)</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 135f7f2ab9b3eec6b79638c6d89b52b5bb60becc
</p>

**Review by clairesunstudio — APPROVED** [review]

File: changelogs/DP-31793.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# You may add more than 1 description & issue for each type using the following format:
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Type:
```
Comment (clairesunstudio) [inline changelogs/DP-31793.yml]:
```suggestion
Changed: 
```

File: changelogs/DP-31793.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - description: Describe the change. If you need multiple lines, start the first line with the following "|-" characters.
+    issue: DP-****
```
Comment (clairesunstudio) [inline changelogs/DP-31793.yml]:
```suggestion
    issue: DP-31793
```

File: changelogs/DP-31793.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Type:
+  - description: Describe the change. If you need multiple lines, start the first line with the following "|-" characters.
```
Comment (clairesunstudio) [inline changelogs/DP-31793.yml]:
```suggestion
  - description: Add outline to page alerts and change icon and color to improve accessibility
```

**Pull request merged**